src/Aspire.Hosting/Dashboard/proto/resource_service.proto

```diff
@@ -131,6 +131,15 @@ message Url {
     // Determines if this url shows up in the details view only by default.
     // If true, the url will not be shown in the list of urls in the top level resources view.
     bool is_internal = 3;
+    // Display properties of the Url
+    optional EndpointDisplayProperties display_properties = 4;
+}
+
+message EndpointDisplayProperties {
```

Comment:
Rename to `UrlDisplayProperties`. The message this is used on is called `Url` so it should follow its naming standard.